src/components/Photo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import 'normalize.css';
+
+function Photo(){
+  return (
+    <li>
+      <img src = {city} />
```

Comment:
```suggestion
      <img src={photo} />
```
Whatever you end up calling this, it has to be consistent throughout the file.